Resolve Error lines due to Typescript Errors

# Cannot Resolve Type Aliases for Components in Editor

## Describe the bug
In the editor, imports using type aliases (e.g., `components/Header`) show unresolved module errors like:  
`Cannot find module 'components/Header' or its corresponding type declarations.ts(2307)`  
Despite this, the build, tests, and dev server work without any issues.

## To Reproduce
Steps to reproduce the behavior:
1. Set up type aliases in `tsconfig.json` for the `components` folder.
2. Use the alias in imports, e.g., `import Header from 'components/Header'`.
3. Observe error lines in the editor (e.g., VS Code).

## Expected behavior
The editor should recognize and resolve type aliases configured in `tsconfig.json`, matching the behavior of the build process.ot


##screenshot

![image (1)](https://github.com/user-attachments/assets/bd210f62-e26a-4078-b39b-83fffd5ff64c)

--- a/frontend/global.d.ts
+++ b/frontend/global.d.ts
@@ -1,0 +1,1 @@
+declare module 'src/*'

--- a/frontend/tsconfig.json
+++ b/frontend/tsconfig.json
@@ -21,5 +21,5 @@
       "*": ["src/*"]
     }
   },
-  "include": ["__tests__", "src", "__tests__/*", "__tests__/src/data"]
+  "include": ["__tests__", "src"]
 }

--- a/frontend/tsconfig.json
+++ b/frontend/tsconfig.json
@@ -21,5 +21,5 @@
       "*": ["src/*"]
     }
   },
-  "include": ["__tests__/*", "__tests__/src/data"]
+  "include": ["__tests__", "src", "__tests__/*", "__tests__/src/data"]
 }

